man/docker-images.1.md

```diff
@@ -103,6 +103,24 @@ which displayed different visualizations of the image data. Docker core removed
 this functionality in the 1.7 version. If you liked this functionality, you can
 still find it in the third-party dockviz tool: https://github.com/justone/dockviz.
 
+## Listing images in a desired format
+
+You can use the `--format` parameter to list only the image attributes required
+and in the order required. For example, if the image ID, Tag, and REPOSITORY 
+are required, in that order, then the following command is used:
+
+    docker images --format "{{.ID}}: {{.Tag}}\t{{.Repository}}"
+
+It produces output similar to :
+
+    11a5207645d4: latest	docker.io/fedora
```

Comment:
Looks like you're running the Red Hat fork of Docker, which prepends `docker.io/` to images from Docker hub; when you update the example, can you remove that prefix, as it's not in the official builds